examples/azure-linux-tdx-vm/variables.tf

```diff
@@ -0,0 +1,64 @@
+########################
+####    Required    ####
+########################
+
+variable "admin_password" {
+  description = "The Password which should be used for the local-administrator on this virtual machine"
+  type        = string
+  sensitive   = true
+  validation {
+    condition     = length(var.admin_password) >= 8
+    error_message = "The admin_password value must be at least 8 characters in length"
+  }
+}
+
+variable "azurerm_resource_group_name" {
+  description = "Name of the resource group to be imported"
+  type        = string
+  default     = "DS-TDXTERRAFORM"
+}
+
+variable "virtual_network_resource_group_name" {
+  description = "Name of the resource group of the virtual network"
+  type        = string
+  default     = "DS-TDXTERRAFORM"
+}
+
+variable "azurerm_virtual_network_name" {
+  description = "Name of the preconfigured virtual network"
+  type        = string
+  default     = "dstdxvnet"
+}
+
+variable "azurerm_subnet_name" {
+  description = "The name of the preconfigured subnet"
+  type        = string
+  default     = "dstdxsubnet"
+}
+
+variable "keyvault_name" {
+  description = "The name of the azure key vault"
+  type        = string
+  default     = "ds-tdx-examplekeyvault"
+}
+
+variable "virtual_machine_size" {
+  description = "The SKU that will be configured for the provisioned virtual machine"
+  type        = string
+  default     = "Standard_DC2es_v5"
+}
+
+variable "vm_name" {
+  description = "The unique name of the Linux virtual machine"
+  type        = string
+  default     = "ds-tdx-linuxvm1"
+}
```

Comment:
For simplicity, delete these, use real values on the example main.tf

Reply:
Done